src/load.ts

```diff
@@ -94,3 +97,8 @@ export async function getDeps(cwd: string) {
 
   return deps
 }
+
+export async function getDepsHash(cwd: string) {
+  const deps = await getDeps(cwd, true)
+  return hash(deps.join(','))
```

Comment:
why not directly compare `deps.join(',')`?

Reply:
I think we also need to rebuild when the dependency version changes

Reply:
Looks good

Reply:
I'll add it later~